### ISSUE TITLE
Issues for dataloader

### ISSUE THREAD

**Issue by XrKang** [KU-CVLAB/DaRF#3]
Thanks for your great work.

I have some questions about your code. 

1. How select the colmap scale of different scenes for depth processing?
2. How select the scale and center to generate poses for rending?
3. How can I revise the dataloader for the LLFF dataset? Can you give me some advice?

Looking forward to your reply! Thank you!

**Comment by seong0905** [comment]
Thanks for your interest in our work.
I will provide answers to each question.

1. We calculated the colmap scale for each scene according to the following code included in the dataset.
https://github.com/KU-CVLAB/DaRF/blob/47b2d1a23d13f0d149e55cf8fd2195ec42093d1e/plenoxels/datasets/scannet_dataset.py#L255-L275

2. For rendering, we normalized the poses from the dataset in a similar manner as described above. If poses are not normalized, both the colmap scale and camera pose can be used as they are in the dataset without considering scale and center adjustments.

3. If you refer to [K-Planes](https://github.com/sarafridov/K-Planes), it should guide you on the LLFF dataloader. And if using LLFF data, it seems that there is no need to consider scale and center adjustments.
